master github-actions tester is broken

I am seeing
```
/__w/_temp/c567fb68-12c8-4a2c-bff5-5a37c02e7d31.sh: 3: cmake: not found
Error: Process completed with exit code 1[27](https://github.com/geodynamics/aspect/actions/runs/9286521577/job/25553408240?pr=5647#step:4:28).
```
with the image geodynamics/aspect-tester:focal-dealii-master
See https://github.com/geodynamics/aspect/actions/runs/9286521577/job/25553408240?pr=5647#step:4:27

@gassmoeller was there a recent change to that docker image?

--- a/contrib/docker/tester/Dockerfile
+++ b/contrib/docker/tester/Dockerfile
@@ -27,7 +27,7 @@ RUN cd /opt && \
 
 # Set environment variables for this image to be used
 # by Github Actions
-ENV PATH="/opt/astyle-2.04:/opt/cmake-3.20.5-linux-x86_64/bin:$PATH"
+ENV PATH="/opt/astyle-2.04:/opt/cmake-3.26.4-linux-x86_64/bin:$PATH"
 ENV DEAL_II_DIR /opt/deal.II-master
 ENV NETCDF_DIR /opt/netcdf-4.7.4
 ENV OMPI_MCA_btl_base_warn_component_unused=0
@@ -38,7 +38,7 @@ ENV OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
 
 # Set environment variables for derived images, which
 # are build by Jenkins
-ONBUILD ENV PATH="/opt/astyle-2.04:/opt/cmake-3.20.5-linux-x86_64/bin:$PATH"
+ONBUILD ENV PATH="/opt/astyle-2.04:/opt/cmake-3.26.4-linux-x86_64/bin:$PATH"
 ONBUILD ENV DEAL_II_DIR /opt/deal.II-master
 ONBUILD ENV NETCDF_DIR /opt/netcdf-4.7.4
 ONBUILD ENV OMPI_MCA_btl_base_warn_component_unused=0

